RockLib.Configuration/RockLibConfigurationBuilderExtensions.cs

```diff
@@ -46,8 +47,32 @@ public static IConfigurationBuilder AddRockLib(this IConfigurationBuilder builde
             builder = builder.AddJsonFile(jsonConfigPath, optional: true);
 
             if (!string.IsNullOrEmpty(supplementalJsonConfigPath))
+            {
                 builder = builder.AddJsonFile(supplementalJsonConfigPath, optional: true);
+            }
+           
+            return builder;
+        }
+
+        /// <summary>
+        /// Adds support for .Net Framework applications to pull in App or Web.config AppSettings values.
+        /// </summary>
+        /// <param name="builder">The <see cref="IConfigurationBuilder"/> to add to.</param>
+        /// <returns>The <see cref="IConfigurationBuilder"/></returns>
+        public static IConfigurationBuilder AddAppOrWebConfig(this IConfigurationBuilder builder)
+        {
+
+#if NET451
```

Comment:
It seems a little weird to put the `#if` inside this method - it doesn't make sense for the method to exist at all in .NET Standard. Should put the `#if` outside this method and around the call to it.

Reply:
I prefer to have this inside the method.  I think it keeps the GetDefaultRoot cleaner.  Also allows us to do something else in the AddAppOrWebConfig if needed in the future for other frameworks.